src/frontend/planner_test/tests/testdata/output/sink.yaml

```diff
@@ -56,16 +56,18 @@
     ├── columns:
     │   ┌── kv_log_store_epoch: bigint
     │   ├── kv_log_store_seq_id: integer
+    │   ├── kv_log_store_vnode: smallint
```

Comment:
How about putting the vnode column as the first colum? Sounds more intuitive and consistent with other usage of vnode column to me.

Reply:
I used to consider this. However, if we put the vnode column as the first column, when we truncate the log store, we will have to write a different table watermark for all vnodes, since their table watermarks get different with the vnode prefix. The newly added vnode column is used as a hint for batch query, and therefore I think it's fine to put it to the last pk column.

Reply:
Currently when we truncate the log store on the latest consumed epoch, we will write `TableWatermark`s to the storage side to delete the records before the consumed epoch. `TableWatermark` means the lower bound within each vnode (See https://github.com/risingwavelabs/risingwave/issues/13148 for details). 

By design, if the table watermark (lower bound) of all vnodes within an executor is the same (like `(v1, wtm1), (v2, wtm1)`), they can be merged to `((v1, v2), (wmt1))`. Now if we add the vnode column to the pk prefix, the watermark will become `(v1, [v1-wtm1]), (v2, [v2-wtm1])`, and their watermark value becomes different and they cannot be merged, which will greatly increase the size of metadata from # of parallelism to # of vnode.